packages/@react-stately/layout/src/ListLayout.ts

```diff
@@ -510,4 +484,38 @@ export class ListLayout<T> extends Layout<Node<T>> implements KeyboardDelegate {
     layoutInfo.transform = 'scale3d(0.8, 0.8, 0.8)';
     return layoutInfo;
   }
+
+  getDropTargetFromPoint(x: number, y: number, isValidDropTarget: (target: DropTarget) => boolean): DropTarget {
```

Comment:
difference between this and listdroptargetdelegate? or reason for both?

Reply:
ListDropTargetDelegate uses the DOM for layout information, ListLayout uses the layout when virtualized